### ISSUE TITLE
Button to toggle Autoplay

### ISSUE THREAD

**Issue by DonovanDarko** [language-transfer/lt-app#76]
As a language learner, I want to be able to listen to one lesson at a time (while not staring at my phone) so that I can reflect on what I've just learned and take the course at my own pace.

**Comment by jonathanPoitz** [comment]
+1

**Comment by xeruf** [comment]
yes please! maybe even as a timer so somebody can say stop after X lessons or X minutes, optionally with breaks in between
